File: frontend/PokeQuest_Game/src/types/Feyling.ts
export type Feyling = {
    id: number,
    name: string,
    description: string,
    img: string,
    typeId: number,
    abilityId: number,
    isUnlocked: boolean,
    hp: number,
    atk: number,
    itemId: number | null,
    weakAgainstId: number,
    strongAgainstId: number,
    sellPrice: number
}

File: wpfapp/PokeQuestWPF/Feyling.cs
public class Feyling
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public string Img { get; set; }
    public int TypeId { get; set; }
    public int AbilityId { get; set; }
    public bool IsUnlocked { get; set; }
    public int Hp { get; set; }
    public int Atk { get; set; }
    public int? ItemId { get; set; }
    public int WeakAgainstId { get; set; }
    public int StrongAgainstId { get; set; }
    public int SellPrice { get; set; }
}
